pkg/kubelet/kuberuntime/kuberuntime_container_linux_test.go

```diff
@@ -21,10 +21,15 @@ package kuberuntime
 import (
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
 	"github.com/stretchr/testify/assert"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why the alias? Same in other places too.

Reply:
> v1 "k8s.io/api/core/v1"

This was added by goimports and I didn't notice. TIL this is intentional: https://github.com/golang/go/issues/30051#issuecomment-495354932